R/shell_connection.R

```diff
@@ -453,26 +453,27 @@ connection_is_open.spark_shell_connection <- function(sc) {
 
 #' @export
 spark_log.spark_shell_connection <- function(sc, n = 100, filter = NULL, ...) {
-  if (!is.null(sc$output_file) && file.exists(sc$output_file)) {
```

Comment:
Consolidate `spark_log.spark_shell_connection()` and `spark_log.spark_connection()`.